docs/sources/developers/http_api/user.md

```diff
@@ -36,6 +36,8 @@ To use these API endpoints you have to use Basic authentication and the Grafana
 The `admin` user that Grafana is provisioned with by default has permissions to use these API endpoints.
 {{< /admonition >}}
 
+For Grafana Cloud customers, refer to [Organization HTTP API](/docs/grafana/latest/developers/http_api/org/) for finding users with the org Admin role.
```

Comment:
These links should use version substitution to make sure that the link always points to the HTTP API for the correct version of docs.

https://grafana.com/docs/writers-toolkit/write/shortcodes/#about-version-substitution